doc/nrf/device_guides/working_with_nrf/nrf52/developing.rst

```diff
@@ -58,6 +58,10 @@ To enable support for FOTA updates, do the following:
    * SMP BT reassembly feature.
    * The :kconfig:option:`CONFIG_NCS_SAMPLE_MCUMGR_BT_OTA_DFU_SPEEDUP` Kconfig option automatically extends the Bluetooth buffers, which allows to speed up the FOTA transfer over Bluetooth, but also increases RAM usage.
 
+.. note::
+   The :kconfig:option:`CONFIG_NCS_SAMPLE_MCUMGR_BT_OTA_DFU` Kconfig option can be used with specific Nordic Semiconductor devices to enable MCUmgr to perform firmware over-the-air (FOTA) updates using Bluetooth LE.
```

Comment:
We're in the nRF52 guide here, so maybe just list the compatible nRF52 Series devices instead of "specific Nordic Semiconductor devices"?